src/lib/eZ/RichText/Resources/stylesheets/xhtml5/edit/docbook.xsl

```diff
@@ -722,11 +722,24 @@
           <xsl:value-of select="@class"/>
         </xsl:attribute>
       </xsl:if>
-      <xsl:if test="@data-ezalign">
-        <xsl:attribute name="ezxhtml:align">
-          <xsl:value-of select="@data-ezalign"/>
+      <xsl:if test="@id">
+        <xsl:attribute name="xml:id">
+          <xsl:value-of select="@id"/>
         </xsl:attribute>
       </xsl:if>
+      <xsl:if test="contains( @style, 'text-align:' )">
+        <xsl:variable name="textAlign">
+          <xsl:call-template name="extractStyleValue">
+            <xsl:with-param name="style" select="@style"/>
+            <xsl:with-param name="property" select="'align'"/>
```

Comment:
Why not narrow this down to the specific rule:
```suggestion
            <xsl:with-param name="property" select="'text-align'"/>
```
?

Reply:
Yeah, that's a good idea.